pkg/deploy/dev-workspace/dev_workspace.go

```diff
@@ -432,11 +432,19 @@ func syncObject(deployContext *deploy.DeployContext, obj2sync *Object2Sync, name
 		return false, err
 	}
 
+	cheNamespaceOwnership := actual.(metav1.Object).GetAnnotations()[deploy.CheEclipseOrgNamespace]
+
+	// don't sync when object exists but it's not managed by Che
+	if exists && cheNamespaceOwnership == "" {
+		return true, nil
+	}
+
 	// sync objects if it has been created by same operator
 	// or it is the only operator with the `spec.devWorkspace.enable: true`
 	if !exists ||
```

Comment:
Could you be more specific and use `()` to emphasis the order of operations?

Reply:
well, it's a base of mathematical logic but I can wrap && into ()